Makefile

```diff
@@ -320,6 +318,7 @@ TESTS = \
 	iostats_context_test \
 	db_wal_test \
 	db_block_cache_test \
```

Comment:
I'm not sure we want to include Travis thing in Makefile. Would that confuse people?